docs/development/ceps/cep-001.rst

```diff
@@ -0,0 +1,243 @@
+.. _cep-001:
+
+
+**********************************
+CEP 1 - Purpose and guidelines
+**********************************
+
+* Status: draft
+* Discussion:  NA
+* Date accepted: NA
+* Last revised: Febuary 10, 2023
+* Author: Tomas Bylund
+* Created: Febuary 10, 2023
+
+Abstract
+========
+
+CEP stands for "Ctapipe Enhancment Proposal". This is CEP 1, which
+describes the purpose of writing CEPs as well as giving some guidelines
+on how CEPs are authored, discussed and reviewed.
+
+An CEP is a design document providing information to the community, or 
+describing a new feature for ctapipe or its processes or environment. 
```

Comment:
only new features? I'd also include larger changes or design decisions

Reply:
changed. 

I think some repetition between the abstract and the introduction is unavoidable, but I now use the statement you like more in both places.